test/compilable/extra-files/depsOutput9948a.d

```diff
@@ -3,4 +3,4 @@ module depsOutput9948a;
 void templateFunc(string myImport)()
 {
    mixin(myImport);
-}
```

Comment:
Then don't make a PR for it.  Either way it should not be in _this_ PR.

In general, enforcing style in the test suite is actually counterproductive - it reduces coverage of code styles that must be supported by the compiler, even if they are discouraged.